pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -1124,7 +1118,7 @@ func (nc *Controller) tryUpdateNodeHealth(ctx context.Context, node *v1.Node) (t
 		for _, nodeConditionType := range nodeConditionTypes {
 			_, currentCondition := controllerutil.GetNodeCondition(&node.Status, nodeConditionType)
 			if currentCondition == nil {
-				klog.V(2).Infof("Condition %v of node %v was never updated by kubelet", nodeConditionType, node.Name)
+				klog.V(2).InfoS("Condition of node was never updated by kubelet", " nodeConditionType", nodeConditionType, "nodeName", node.Name)
```

Comment:
extra space " nodeConditionType"

Reply:
Yep, good catch. I wonder whether the logcheck tool should have additional sanity checks for the content of the key string...

Reply:
sorry, I have updated it